### PR TITLE
Support run on player build [experimental]

### PR DESCRIPTION
**Pull request by nowsprinting**
### Overview

Added support for running player builds.
It is needed for performance testing, aging, etc.

It's still unfriendly and time-consuming.
So, I've made it an experimental feature.

Please see the README for the differences in specifications.

### Main changes

- Fix to Launcher to run on the player build
    - Fix tests run on player
    - Refactor around launch/terminate
- Fix file output path on the player
- Add command line argument for launch autopilot on player

#### Chores, fix bugs

- Fix survive agents after terminate
- Fix reset trigger on OneTimeAgent
- Fix InitializeOnLaunchAutopilotAttribute inherit PreserveAttribute
- Fix tests
- Fix READMEs
- Remove not used asset files

### Priority

I'd want it to be released within 1 to 2 weeks.

---

### Contribution License Agreement

- [x] By placing an "x" in the box, I hereby understand, accept and agree to be bound by the terms and conditions of the [Contribution License Agreement](https://dena.github.io/cla/).

### CODE REVIEW

**Comment by nowsprinting** [comment]
I will rebase this PR after the merge #77

**Comment by nowsprinting** [comment]
rebased!

**Review by bo40 — DISMISSED** [review]
I made a few minor comments.
Overall, it looks good to me.

File: Runtime/Autopilot.cs [hunk 1]
```diff
@@ -181,18 +182,17 @@ public async UniTask TerminateAsync(ExitCode exitCode, string logString = null,
...
+            // Terminate when launch from edit mode (including launch from commandline)
             _logger.Log("Stop playing by autopilot");
-            _state.exitCode = exitCode;
             // XXX: Avoid a problem that Editor stay playing despite isPlaying get assigned false.
             // SEE: https://github.com/DeNA/Anjin/issues/20
-            await UniTask.DelayFrame(1, cancellationToken: token);
+            await UniTask.NextFrame(token);
             EditorApplication.isPlaying = false;
-            // Call Launcher.OnChangePlayModeState() so terminates Unity editor, when launch from CLI.
+            // Note: Call `Launcher.OnChangePlayModeState()` so terminates Unity editor, when launch from commandline.
```
[the remark this answers is not in the document]
Comment (nowsprinting) [reply Runtime/Autopilot.cs]: Thanks!
Fixed in 5e9ed07.

File: Editor/Commandline.cs [hunk 2]
```diff
@@ -98,5 +101,27 @@ private static void FocusGameView()
             var gameView = assembly.GetType(viewClass);
             EditorWindow.GetWindow(gameView, false, null, true);
         }
+
+        /// <summary>
+        /// Stop autopilot on play mode exit event when run on Unity editor.
+        /// Not called when invoked from play mode (not registered in event listener).
+        /// </summary>
+        /// <param name="playModeStateChange"></param>
+        /// <exception cref="ArgumentOutOfRangeException"></exception>
```
[the remark this answers is not in the document]
Comment (nowsprinting) [reply Editor/Commandline.cs]: Thanks!
Fixed in e8e8b27.

**Review by bo40 — APPROVED** [review]
LGTM

**Comment by bo40** [comment]
Once #83 and #85 are merged, we plan to release this feature as v1.7.0.

**Pull request merged**